.github/workflows/nethermind-tests.yml

```diff
@@ -77,8 +77,6 @@ jobs:
     steps:
     - name: Check out repository
       uses: actions/checkout@v3
-      with:
-        submodules: true
```

Comment:
I don't think we don't need this.
A new test suite is added but not included below. Hence, it doesn't run, hence, no error. See around the line 140.

Reply:
That step is part of the ethereum tests action